server/stt_metrics_test.go

```diff
@@ -0,0 +1,70 @@
+package server
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	managementClient "github.com/percona/pmm/api/managementpb/json/client"
+	serverClient "github.com/percona/pmm/api/serverpb/json/client"
+	"github.com/percona/pmm/api/serverpb/json/client/server"
+	"github.com/prometheus/client_golang/api"
+	promapi "github.com/prometheus/client_golang/api/prometheus/v1"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	pmmapitests "github.com/Percona-Lab/pmm-api-tests"
+)
+
+const promAddress = "http://admin:admin@127.0.0.1:80/prometheus"
```

Comment:
That is configurable in https://github.com/Percona-Lab/pmm-api-tests/blob/master/init.go, please use BaseURL with a custom path. Something like
```
pmmapitests.BaseURL.ResolveReference(&url.URL{
  Path: "/prometheus",
})
```

(there should be a similar code somewhere)

Reply:
that would change the URL to `"https://admin:admin@127.0.0.1:443/prometheus"` and that was making my tests fail locally

Reply:
i will push it and see if it fails on the CI too